_drafts/oryx/technology.md

```diff
@@ -0,0 +1,31 @@
+# Oryx technologies
+
+Oryx is a frontend framework that is designed to help developers build web applications using modern technologies.
+It is built on a variety of technologies and in this article, we will take a closer look at each of these technologies and see how they contribute to the development of Oryx.
```

Comment:
I'd emphasize that:
- Oryx uses standards wherever available and applicable. This is true for the technologies and tools
- Oryx tries to use the least necessary build tools and likes to stick to the "native" web where possible.
- Oryx aims to use JavaScript during development, but tries avoid the need to run JavaScript in the browser. Wherever possible, html and CSS are preferred to render a fast loading experience cross different screen sizes